packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -103,6 +103,12 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
    */
   disabled?: boolean;
 
+  /**
+   *  Optional flag to decide that thumb will snap to closest value onChange.
+   */
+
+  snaps?: boolean;
```

Comment:
how about using steps approach instead? Similar to [material](https://material-ui.com/components/slider/#range-sliders) it could support `steps=1` to support what snaps does, as well as `steps=2` for even numbers, or an array [0,32,100] to snap to specific points.

Reply:
Slider already has steps... https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts#L68 Is this what you meant or something else?

Reply:
ah, if we already have that prop, a different name for sure. it looks like you just added marks which is the same idea. Good!

Reply:
Hey I'm actually going to add marks in a subsequent PR along with tooltips for thumblabel

Reply:
I called this `snapToStep` here: https://codesandbox.io/s/composed-prototype-zovck

Reply:
Also the comment should be clear that this really only applied to mouse drags. By default on mouse drags, we allow the user to gracefully drag across the slider, though value should only snap to a step. When they let go, the thumb moves into position. This flag persists the snapping behavior.

Reply:
@dzearing do you want me to change the comments and change the property name from snaps to snapToStep?